examples/olives/package.json

```diff
@@ -1,10 +1,17 @@
 {
   "private": true,
   "dependencies": {
-    "olives": "^1.6.0",
-    "emily": "^1.8.1",
-    "requirejs": "^2.1.5",
-    "todomvc-common": "^1.0.1",
-    "todomvc-app-css": "^1.0.1"
+    "emily": "^3.0.3",
+    "olives": "^3.0.5",
+    "todomvc-app-css": "^1.0.1",
+    "todomvc-common": "^1.0.1"
+  },
+  "scripts": {
+    "build": "browserify ./js/app.js -o olives-todo.js",
+    "watch": "watchify ./js/app.js -o olives-todo.js"
+  },
```

Comment:
Mind adding a note about these scripts in the `examples/olives/readme.md`? Thanks!